conf/modules.config

```
@@ -76,11 +111,56 @@ process {
         ]
     }
 
+    withName: 'DROP.*OUTRIDER_RESULTS' {
+        cpus = { 3 * task.attempt }
+        memory = { 30.GB * task.attempt }
+        time = { 1.h * task.attempt }
+    }
+
     //
     // Aberrant splicing
     //
 
+    withName: 'DROP.*SPLICECOUNTS_INIT' {
+        cpus = { 2 * task.attempt }
+        memory = { 5.GB * task.attempt }
+        time = { 1.h * task.attempt }
+    }
+
+    withName: 'DROP.*SPLICECOUNTS_COUNTSPLITREADS' {
+        cpus = { 3 * task.attempt }
+        memory = { 10.GB * task.attempt }
+        time = { 1.h * task.attempt }
+    }
+
+    withName: 'DROP.*SPLICECOUNTS_MERGESPLITREADS' {
+        cpus = { 20 * task.attempt }
+        memory = { 70.GB * task.attempt }
+        time = { 1.h * task.attempt }
+    }
+
+    withName: 'DROP.*SPLICECOUNTS_COUNTNONSPLITREADS' {
+        cpus = { 3 * task.attempt }
+        memory = { 10.GB * task.attempt }
+        time = { 1.h * task.attempt }
+    }
+
+    withName: 'DROP.*SPLICECOUNTS_MERGENONSPLITREADS' {
+        cpus = { 20 * task.attempt }
+        memory = { 30.GB * task.attempt }
+        time = { 1.h * task.attempt }
+    }
+
+    withName: 'DROP.*SPLICECOUNTS_COLLECTCOUNTS' {
+        cpus = { 2 * task.attempt }
+        memory = { 10.GB * task.attempt }
+        time = { 1.h * task.attempt }
+    }
+
     withName: 'DROP.*FRASER_FILTEREXPRESSION' {
+        cpus = { 3 * task.attempt }
+        memory = { 30.GB * task.attempt }
```

maybe a bit more mem?
```suggestion
        memory = { 40.GB * task.attempt }
```